src/components/BooksComponents/BookItem.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const BookItem = () => (
+  <div>
+    <h3>Book title</h3>
+    <p>Category</p>
+    <button type="button">Delete</button>
+  </div>
+);
```

Comment:
- [ ] It will be great if the `bookItem` takes in book data as a prop and  renders the title and the author. This is a requirement for this project.Below is a hint on how you can solve this issue ⬇️ 

1.  In your `bookList` component where you are rendering the `bookItem` component, you can create some dummy data of books since we don't have the data from the API yet. Remember the point is to make your component ready for the data it will receive. You can do something like 
```
    const bookArray = [ 
      {id: 1, title: bookName, author: book author },
    {id: 2, title: bookName, author: book author },
]
```
2. Then map through the array and for each item pass the data as props to the `bookItem` component
3. Then in your `bookItem` component receive the props and destructure the data to be used in the component